src/js/tracks/text-track-cue-list.js

```diff
@@ -12,72 +13,93 @@ import document from 'global/document';
  *   getter TextTrackCue (unsigned long index);
  *   TextTrackCue? getCueById(DOMString id);
  * };
+ *
+ * @param {Array} cues cues to add to the list
```

Comment:
This comment also needs to be updated. `cues to be initialized with` or something.